rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -284,6 +284,7 @@ Requires: pcre
 # -L adds the library path to the produced executables
 # -static-libgcc is added because gcc is not provided on aix
 %define build_ldflags -static-libgcc -Wl,-brtl -Wl,-L%{rudderdir}/lib
+%define build_cflags -static-libgcc
```

Comment:
shouldn't it be defined in the %else too ?

Reply:
it should, with an empty value

Reply:
`%{nil}`